ArticleTemplates/assets/js/modules/creativeInjector.js

```diff
@@ -13,13 +13,27 @@ function (
         if (!initialised) {
             initialised = true;
 
-            window.injectCreative = injectCreative;
-            window.applyNativeFunctionCall('injectInlineCreative');
+            if (GU.opts.contentType === 'liveblog' || document.querySelector('.article__body--liveblog')) {
+              trackLiveBlogEpic();
+            } else {
+              window.injectCreative = injectCreative;
+              window.applyNativeFunctionCall('injectInlineCreative');
+            }
+        }
+    }
+
+    function trackLiveBlogEpic() {
+        var liveBlogEpicContainer = document.querySelector('contributions-epic');
+
+        if (liveBlogEpicContainer) {
+            var liveBlogEpicContainerId = liveBlogEpicContainer.getAttribute('id')
+
+            isCreativeInView(liveBlogEpicContainer, liveBlogEpicContainerId)
```

Comment:
@GHaberis sorry I must have misunderstood your last comment for this. I will revert